e2e/iam/project_api_keys_test.go

```diff
@@ -51,11 +54,21 @@ func TestProjectAPIKeys(t *testing.T) {
 			if err := json.Unmarshal(resp, &key); err != nil {
 				t.Fatalf("unexpected error: %v", err)
 			}
-			a.Equal("e2e-test", key.Desc)
+			a.Equal(desc, key.Desc)
 			ID = key.ID
 		}
 	})
 
+	if ID == "" {
+		assert.FailNow(t, "Failed to create API key")
+	}
+
+	defer func() {
```

Comment:
so if you have a project API key it's still bound to the org even if removed from a project